src/simtools/schemas/model_parameter.metaschema.yml

```diff
@@ -25,14 +25,19 @@ definitions:
         type: boolean
         description: "This parameter is a file."
       instrument:
-        type: string
+        type:
+          - string
+          - "null"
         description: "Associated instrument."
       site:
-        type: string
+        type:
+          - string
+          - "null"
         description: "Associated CTAO site."
         enum:
           - North
           - South
+          - null
```

Comment:
If we accept null here: Could this result in some conflicts where we missed to add the site?

Reply:
Site always needs to be presents, it is part of the list of required parameters (further down in the file):

```
   required:
      - file
      - instrument
      - parameter_version
      - schema_version
      - site
      - type
      - unit
      - value
```

Reply:
Ok missed this one. All good then.